en/docs/develop/integration-artifacts/file/ftp-sftp.md

```
@@ -1,166 +1,648 @@
+</Tabs>
+
+### SSL/TLS configuration (FTPS)
+
+FTPS encrypts the control and data channels using SSL/TLS. In addition to `auth`, pass a `secureSocket` record to the listener to control certificate verification and key material.
+
+<Tabs>
+<TabItem value="ui" label="Visual Designer" default>
+
+In the **Configure** panel, expand **Advanced Configurations** and enter the `secureSocket` record under **Configuration for ftpsListener**:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)